app/routes/contacts/overview/mailing-lists.js

```diff
@@ -1,4 +1,15 @@
 import Route from '@ember/routing/route';
+// eslint-disable-next-line ember/no-mixins
+import DataTableRouteMixin from 'ember-data-table/mixins/route';
 
-export default class ContactsOverviewMailingListsRoute extends Route {
-}
+export default class ContactsOverviewMailingListsRoute extends Route.extend(DataTableRouteMixin) {
+  modelName = 'contact-list';
+
+  mergeQueryOptions(params) {
+    const queryParams = {
+      sort: params.sort
+    };
+
+    return queryParams;
+  }
```

Comment:
`sort` zit by default al in de query params die naar de backend gestuurd worden via de `DataTableRouteMixin`. Deze `mergeQueryOptions` mag hier dus volledig verwijderd worden.